lib/LedgerSMB/Template.pm

```diff
@@ -410,7 +410,7 @@ sub new_UI {
             if $LedgerSMB::App_State::Company_Config;
     my $UI_vars = {
         dojo_theme => $dojo_theme // $LedgerSMB::Sysconfig::dojo_theme,
-        dojo_built => $LedgerSMB::Sysconfig::dojo_built,
```

Comment:
I'd like to leave dojo_built available to js if possible

Reply:
Sorry, we agreed on that and I forgot.